### DISCUSSION TITLE
Timelapse issue

### DISCUSSION THREAD

**Discussion by Espusen** [gopro/labs#936] [General]
Hi David
I have been running a previously tested script. 
Today it seemed to get stuck after each image was shot. The text "Labs Processing" kept flashing at the top of the screen with no shutdown occurring until the next image was scheduled to take.
I might have coinsided with me plugging in a PowerBank. 
Thoughts?
https://drive.google.com/file/d/1E_7XCaKv9SdJ6C6WNr6PCbkf7Q0D9kKd/view?usp=drive_link

**Comment by dnewman-gpsw** [comment]
Were the image between 9:38 and 11:45 captured? If it is was otherwise working, just not sleeping, the photo captures are taking longer to close than the script expected. This could be the SD card folder is filing, but I don't have a strong theory on this yet. There might be is a bug fix for this coming, the next firmware will wait until the file has closed (rather than assume it had.) The workaround:

```mPdP>08:59<17:00!S+!1N+!599RQ!3600RQ```

**Comment by Espusen** [comment]
Sorry I thought I replied.
Yes the images captured, but camera did not shutdown between images which is the point of course.
SD card was 2% full. (Sandisk U3) 128Gb

New code has worked for 3 days.

**Comment by dnewman-gpsw** [comment]
The new script will be a reliable substitute, until the official fix.

**Reply by Espusen** [reply]
Ok. Works fine so thank you!!
